flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStGeneralMultiGetOperation.java

```diff
@@ -40,32 +44,163 @@ public class ForStGeneralMultiGetOperation implements ForStDBOperation {
 
     private final Runnable subProcessFinished;
 
+    private final Boolean isRemoteDb;
+
+    private final int readIoParallelism;
+
     ForStGeneralMultiGetOperation(
             RocksDB db, List<ForStDBGetRequest<?, ?, ?, ?>> batchRequest, Executor executor) {
-        this(db, batchRequest, executor, null);
+        this(db, batchRequest, executor, null, false, 1);
     }
 
     ForStGeneralMultiGetOperation(
             RocksDB db,
             List<ForStDBGetRequest<?, ?, ?, ?>> batchRequest,
             Executor executor,
-            Runnable subProcessFinished) {
+            Runnable subProcessFinished,
```

Comment:
How about making `subProcessFinished` as the last parameter? More align with other requests' constructor style.